registry/import.go

```diff
@@ -136,6 +143,9 @@ func Import(reader io.Reader) (err error) {
 	}()
 	tw := tar.NewReader(zw)
 
+	pool := threadpool.New(10, 100)
+	s := pool.NewJobGroup()
```

Comment:
By curiosity, why `s` for this variable?

Reply:
Swift, but already taken by package :rofl:

Reply:
Maybe `swiftGroup` in that case